Mathlib/Tactic/Positivity/Basic.lean

```diff
@@ -22,41 +23,70 @@ section Nonsense
 open Function
 -- TODO: these classes are mostly nonsense stubs which should be replaced by the real things
 -- when the theory files are ready
--- FIXME: remove this when the sorries are gone
-set_option warningAsError false
```

Comment:
:tada:

Reply:
There shouldn't be actual proofs in this file, other than one line proofs referencing other theorems and maybe rewriting a bit. These theorems presumably already exist somewhere in mathlib; please relocate them there and import them in this file.

Reply:
Most of these proofs/instances rely on `Algebra.Order.Ring.Defs`, which in turn rely on precisely the file this PR is about. So I hope these proofs will be removed after-the-port of those files. Is that possible, or would you prefer this file is never touched at all? Basically, I'm trying to make sure that this branch has no build CI issues.

Reply:
It's fine if you have a plan to remove these proofs later. I just don't want them to be forgotten, so don't remove the `TODO` at least until then.